### ISSUE TITLE
Panic 6.4.0 on VM tango 9.2

### ISSUE THREAD

**Issue by aurbence** [tango-controls/PANIC#27]
I try PANIC in VirtualMachine from tango-controls website.
but i'm not able to create new alarm.
when i click on new buton i have no new window
and the terminal return : 

```
In AlarmFormula.initStyle(None)
AlarmFormula.initStyle(None) finished.
In AlarmFormula.onEdit(False)
********************************************************************************
<fandango.qt.QDropTextEdit object at 0x7ffaedf80640> => 
In AlarmFormula.updateFormula(,changed=False)
********************************************************************************
<fandango.qt.QDropTextEdit object at 0x7ffaedf80640> => 
In AlarmFormula.updateFormula(,changed=False)
AlarmForm.setCurrentAlarm(Alarm(::0))
In AlarmFormula.onEdit(False)
********************************************************************************
<fandango.qt.QDropTextEdit object at 0x7ffaedf80640> => 
In AlarmFormula.updateFormula(,changed=False)
********************************************************************************
<fandango.qt.QDropTextEdit object at 0x7ffaedf80640> => 
In AlarmFormula.updateFormula(,changed=False)
onNew()
AlarmForm.clearAlarmData()
AlarmForm.setCurrentAlarm(Alarm(::0))
clearing form ...
In AlarmFormula.onEdit(True)
********************************************************************************
<fandango.qt.QDropTextEdit object at 0x7ffaedf80640> => 
In AlarmFormula.updateFormula(,changed=False)
********************************************************************************
<fandango.qt.QDropTextEdit object at 0x7ffaedf80640> => 
In AlarmFormula.updateFormula(,changed=False)
In prepareDataWidget()
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/panic-6.4.0-py2.7.egg/panic/gui/actions.py", line 158, in onNew
    form.onNew()
  File "/usr/local/lib/python2.7/dist-packages/panic-6.4.0-py2.7.egg/panic/gui/editor.py", line 143, in onNew
    self.enableEditForm(True)
  File "/usr/local/lib/python2.7/dist-packages/panic-6.4.0-py2.7.egg/panic/gui/editor.py", line 329, in enableEditForm
    self._tvl.updateStyle()
  File "/usr/local/lib/python2.7/dist-packages/panic-6.4.0-py2.7.egg/panic/widgets.py", line 114, in updateStyle
    obj = self.alarm #obj = self.getModelValueObj()
AttributeError: 'AlarmValueLabel' object has no attribute 'alarm'

```

**Comment by aurbence** [comment]
OK solved i update PANIC version to 6.5.

**Comment by sergirubio** [comment]
Thanks for the feedback anyway!
